tools/trace_reader/main.cc

```diff
@@ -188,9 +190,9 @@ int main(int argc, char* argv[])
 {
   KnobsContainer* knob_container = new KnobsContainer();
   g_knobs = knob_container->getAllKnobs();
-  knob_container->applyParamFile("params.in");
+  // knob_container->applyParamFile("params.in");
```

Comment:
why comment this and below out?

Reply:
@hyojongk Because @hyesoon fixed trace_reader by not reading the params.in file. According to Trello, it needs to be fixed later as a bug report

Reply:
yes for now, just comment out for reading params.in file. now that is causing a seg fault. to be fixed in future.

Reply:
okay am resolving this then